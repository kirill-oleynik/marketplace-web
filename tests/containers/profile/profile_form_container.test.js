const React = require('react');
const renderer = require('react-test-renderer');
const { ProfileFormContainer } = require('../../../app/containers/profile/profile_form_container');

jest.mock('../../../app/components/profile/profile_form', () => 'ProfileForm');

describe('#render', () => {
  test('it renders correctly', () => {
    const tree = renderer.create(
      <ProfileFormContainer
        currentUser={{}}
        errors={{}}
        t={() => {}}
        updateProfile={() => {}}
      />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
