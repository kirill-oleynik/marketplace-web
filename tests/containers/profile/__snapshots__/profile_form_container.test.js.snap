// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<ProfileForm
  currentUser={Object {}}
  errors={Object {}}
  initialState={
    Object {
      "email": "",
      "firstName": "",
      "jobTitle": "",
      "lastName": "",
      "organization": "",
      "phone": "",
    }
  }
  onSubmit={[Function]}
  rules={
    Object {
      "email": Array [
        Array [
          [Function],
          undefined,
        ],
        Array [
          [Function],
          undefined,
        ],
      ],
      "firstName": Array [
        Array [
          [Function],
          undefined,
        ],
        Array [
          [Function],
          undefined,
        ],
      ],
      "jobTitle": Array [
        Array [
          [Function],
          undefined,
        ],
        Array [
          [Function],
          undefined,
        ],
      ],
      "lastName": Array [
        Array [
          [Function],
          undefined,
        ],
        Array [
          [Function],
          undefined,
        ],
      ],
      "organization": Array [
        Array [
          [Function],
          undefined,
        ],
        Array [
          [Function],
          undefined,
        ],
      ],
      "phone": Array [
        Array [
          [Function],
          undefined,
        ],
        Array [
          [Function],
          undefined,
        ],
        Array [
          [Function],
          undefined,
        ],
      ],
    }
  }
  t={[Function]}
  updateProfile={[Function]}
  validateOnChange={true}
  validateSingle={true}
/>
`;
